src/Chainweb/Miner/Coordinator.hs

```diff
@@ -263,19 +256,6 @@ publish lf (MiningState ms) cdb bh = do
             lf Info . JsonLog $ OrphanedBlock (ObjectEncoded bh) (ObjectEncoded p) now mnr msg
         Right r -> lf Info $ JsonLog r
 
--- | The estimated per-second Hash Power of the network, guessed from the time
--- it took to mine this block among all miners on the chain.
---
-estimatedHashes :: PrevTime -> BlockHeader -> Natural
-estimatedHashes (PrevTime p) b = floor $ (d % t) * 1000000
-  where
-    t :: Integer
-    t = case timeBetween (_blockCreationTime b) p of Micros t' -> int t'
-
-    d :: Integer
-    d = case targetToDifficulty $ _blockTarget b of
-        HashDifficulty (PowHashNat w) -> int w
-
```

Comment:
Can you say why this went away?

Reply:
It was used to estimate hash power when we were in the early days of CPU mining, before we had better tools like the Block Explorer. This is only consumed on Kibana, and even then only on one widget that isn't useful.

Reply:
Cool, thanks!